src/test/java/org/folio/tlib/postgres/PgCqlQueryTest.java

```diff
@@ -0,0 +1,125 @@
+        { "Title=men's room", ftResponse("title", "men''s room") },
+        { "Title=v1*", "error: Masking op * unsupported for: Title = v1*" },
+        { "Title=v1?", "error: Masking op ? unsupported for: Title = v1?" },
+        { "Title=v1^", "error: Anchor op ^ unsupported for: Title = v1^" },
+        { "Title=a\\*b", ftResponse("title", "a*b") },
+        { "Title=a\\^b", ftResponse("title", "a^b") },
+        { "Title=a\\?b", ftResponse("title", "a?b") },
+        { "Title=a\\?b", ftResponse("title", "a?b") },
+        { "Title=a\\n", ftResponse("title", "a\\n") },
+        { "Title=\"a\\\"\"", ftResponse("title", "a\"") },
+        { "Title=\"a\\\"b\"", ftResponse("title", "a\"b") },
+        { "Title=a\\12", ftResponse("title", "a\\12") },
+        { "Title=a\\\\", ftResponse("title", "a\\") },
+        { "Title=a\\'", ftResponse("title", "a\\''") },
+        { "Title=a\\'b", ftResponse("title", "a\\''b") },
+        { "Title=a\\\\\\n", ftResponse("title", "a\\\\n") },
+        { "Title=a\\\\", ftResponse("title", "a\\") },
+        { "Title=aa\\\\1", ftResponse("title", "aa\\1") },
+        { "Title=ab\\\\\\?", ftResponse("title", "ab\\?") },
+        { "Title=\"b\\\\\"", ftResponse("title", "b\\") },
+        { "Title=\"c\\\\'\"", ftResponse("title", "c\\''") },
+        { "Title=\"c\\\\d\"", ftResponse("title", "c\\d") },
+        { "Title=\"d\\\\\\\\\"", ftResponse("title", "d\\\\") },
+        { "Title=\"x\\\\\\\"\\\\\"", ftResponse("title", "x\\\"\\") },
+        { "Title=\"\"", "title IS NULL" },
+        { "Title<>\"\"", "title IS NOT NULL" },
+        { "Title==\"\"", "title = ''" },
+        { "Title==\"*?^\"", "title = '*?^'" },
+        { "Title==\"\\*\\?\\^\"", "title = '\\*\\?\\^'" },
+        { "Title==\"b\\\\\"", "title = 'b\\'" },
+        { "Title==\"c\\\\'\"", "title = 'c\\'''" },
+        { "Title==\"d\\\\\\\\\"", "title = 'd\\\\'" },
+        { "Title==\"e\\\\\\\"\\\\\"", "title = 'e\\\"\\'" },
+        { "Title>\"\"", "error: Unsupported operator > for: Title > \"\"" },
+        { "Title==v1 or title==v2",  "(title = 'v1' OR title = 'v2')"},
+        { "isbn=978-3-16-148410-0", "isbn = '978-3-16-148410-0'" },
+        { "isbn=978-3-16-148410-*", "isbn = '978-3-16-148410-*'" },
+        { "cql.allRecords=1 or title==v1", null },
+        { "title==v1 or cql.allRecords=1", null },
+        { "Title==v1 and title==v2", "(title = 'v1' AND title = 'v2')" },
+        { "Title==v1 and cql.allRecords=1", "title = 'v1'" },
+        { "cql.allRecords=1 and Title==v2", "title = 'v2'" },
+        { "Title==v1 not title==v2", "(title = 'v1' AND NOT title = 'v2')" },
+        { "cql.allRecords=1 not title==v2", "NOT (title = 'v2')" },
+        { "title==v1 not cql.allRecords=1", "FALSE" },
+        { "title==v1 prox title==v2", "error: Unsupported operator PROX" },
+        { "cost=1", "cost=1" },
+        { "cost=+1.9", "cost=+1.9" },
+        { "cost=-1,90", "cost=-1,90" },
```

Comment:
`1,90` is invalid SQL

Reply:
Sure, but it's not the job of this module to rigorously parse numbers for all valid formats and reject all invalid ones. This is close enough.

Reply:
Why does the module _explicitly_ ([`case ','`](https://github.com/folio-org/mod-eusage-reports/pull/45/files#diff-ce87b30855c5f29d38f127c9049d6814c54e61a29a2dd7e3b81662bcfd373c37R215)) accept commas when there can never be a comma in SQL numbers?

Reply:
I wasn't sure, so I let it through. I'll change that. Again accepting too much is not a huge problem as long as it does not lead to SQL injection. By the way, what about e  or E.